script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_9B.txt

```diff
@@ -46,94 +46,94 @@ The man concentrates the cold waves flooding out of himself into a single point,
-Without interrupting my thoughts, I let the knife slide automatically the moment I see it.
+Rather than focusing only on Vlov, I observe his surroundings as well.
+Without interrupting my thoughts, I let the knife slide automatically the moment I see something.
 <Page19>
 ―――During the process, I revise the conjecture that I had made earlier.
 That cold wave is spreading in all directions.
 But is the density really as uniform as I thought?
 <Page20>
-The fact that the floor is the coldest means that there is a difference in temperature even within the cold wave. There's the matter of whether the wave is wholly strong or weak, but more importantly, there is a weakness.
+The fact that the floor is the coldest area means that there is a difference in temperature even within the cold wave. There's still the matter of whether the wave is wholly strong or weak, but importantly, it's not perfect.//　地表が最も低温だという事は、寒波の中にも温度差はある。強いか弱いかの基準ではあるが欠点は存在する。
 It may protect him from all directions, but that doesn't mean it makes him invincible all around―――
 <Page21>
 "...So you protect your master on the brink of death. How brave, but futile."
-The long lance that struck down Arcueid gets picked up from the shadows once again.
-Perhaps that was him being spurred on, because behind him, a remarkably large ice tree sprouts branches―――
+The long lance that struck down Arcueid is drawn from the shadows once again.
+Perhaps he was just warming up before, because behind him, a remarkably large ice tree sprouts branches―――//'Priming the pump' それが呼び水になったのか、ヤツの背後にひときわ大きな樹氷が枝を張り―――
 "―――Now. I will release, in great numbers...!"
 <Page22>
-I break my train of thought and pour my focus into intercepting his attack.//text overflow, needs rephrasing
+I put my thoughts on hold and pour all of my attention into intercepting his attack.//Do not touch this page without playtesting
 This is impossible. I can't stop them all.
-There is always a missile coming from somewhere. ―――I won't allow them near me.
+I'll surely be hit somewhere. ―――No.//必ずどこかに被弾する。―――許されない。
 I can't be so naive.
 I can't even let them graze me.
 If I bleed out, it's over. The cold air will not spare even a single minute before coming for my life.
 That's why I must kill. That's why I must struggle.
-I clear my mind and look down only at the 'line'.
+I clear my mind and look only at the 'lines'.//俯瞰 but not sure how to phrase looking only at the lines while also commanding a large-scale view
 <Page23>
 The total number of prey deployed is twenty-six.
-I continue to stare as I develop a strategy for what I can cut first and what I can avoid.
+I continue to stare as I develop a strategy for what I should cut first and what I can avoid.
 ―――Within that gap, I can see it clearly.
 The place of death greater than a line.
-That spot must be the original point of death that gives birth to the lines.
+That spot must be the focal point of death that gives birth to the lines.//あの点こそが、線を生み出す死の極点だと。
```

Comment:
is this right? 極点 doesn't really read to me like focal point at all. Moreover, first time I read the english, I thought it meant like "the primary concern of death" since that's usually how I see that expression. It's not immediately clear it's talking like lens focus.

Reply:
Hmm. What about `locus`?

Reply:
Well, "locus" has the advantage of being a word I've never seen outside of a technical context and thus being _very_ Nasu. Otherwise I don't think I like it... I don't have a better alternative though so I say send it